python/llm/dev/benchmark/all-in-one/run-spr.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-
+source bigdl-nano-init -c
```

Comment:
Do we need `nano` besides `bigdl-nano-init`? If not, we can simple copy the script to `llm` and eliminate the dependency on `nano`

Reply:
Sure, maybe we can add a script named `bigdl-llm-init` and exclude jemalloc related part.